ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/task/AbstractEthTask.java

```diff
@@ -29,19 +31,27 @@
 
 public abstract class AbstractEthTask<T> implements EthTask<T> {
 
+  static volatile LabelledMetric<OperationTimer> ethTasksTimer = null;
```

Comment:
This shouldn't be static - we don't want the timer to be shared across separate instances of Pantheon.  To avoid creating multiple metrics with the same name (which isn't allowed) you'd have to cache them in the MetricsSystem.